### DISCUSSION TITLE
minimize the osg app dynamically

### DISCUSSION THREAD

**Discussion by CapTainHbb** [openscenegraph/OpenSceneGraph#1216] [Q&A]
is there a way to minimize the osg app dynamically? i know we can minimize the osg app somehow by using x11 event but i think it is so complicated to do such operation.

**Comment by robertosfield** [comment]
I just had a look at the GraphicsWindow API and GraphicsWindowX11 and can't see any support for explicitly minimizing or maximizing the window so I think you'll have to tell the X11 window directly to minimize and let the OSG handle this.
